bears/general/CPDBear.py

```diff
@@ -102,6 +108,8 @@ def run(self, language: str, minimum_tokens: int=20,
 
         stdout_output, _ = run_shell_command(arguments)
 
+        os.unlink(temp_filepath)
```

Comment:
Alternatively, you could use `with ... as ...:` :) (Not completely sure if it exists though)